### PR TITLE
PMM-5403 [FB] PMM2 Ubuntu 20.04 support

### PR DESCRIPTION
**Pull request by Sevlin**
Ubuntu 20.04 support for PMM2 client
---
Dependent PR's:
- [x] [jenkins-pipelines](https://github.com/Percona-Lab/jenkins-pipelines/pull/403)

FB test pipeline: [pmm.cd](https://pmm.cd.percona.com/job/ubuntu2004-pmm2-client/)

### CODE REVIEW

File: build/bin/build-client-rpm [hunk 1]
```diff
@@ -14,18 +14,26 @@ main() {
...
+            apt update
+            apt install -y sudo lsb-release
+        fi
+
         export pmm_version=$pmm_version
         export pmm_release=$pmm_release
         export sudo_path=\$(ls /usr/bin/sudo)
         mkdir -p /tmp/pmm
-        pushd /home/builder/results
+        cd /home/builder/results
```
Comment (puneet0191) [inline build/bin/build-client-rpm:29]: why remove pushd pop?

Comment (Sevlin) [reply build/bin/build-client-rpm:29]: The've updated `dash` to make it more strict for POSIX commands. When it called as `/bin/sh` it doesn't support `pushd`/`popd`. These internal commands are only available when binary is called as `/bin/bash`.

Comment (puneet0191) [reply build/bin/build-client-rpm:29]: okay thank you :+1:

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-811-af0f85c
client docker - perconalab/pmm-client-fb:PR-811-af0f85c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-811-af0f85c.tar.gz

File: build/bin/build-client-packages [hunk 2]
```diff
@@ -260,7 +260,7 @@ install_deps() {
         yum -y install git wget rpmdevtools bison yum-utils rpm-build
         install_golang
     else
-        add_percona_apt_repo
+        #add_percona_apt_repo
```
Comment (EvgeniyPatlan) [inline build/bin/build-client-packages:263]: why not just remove it?

Comment (Sevlin) [reply build/bin/build-client-packages]: done

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-811-77d2783
client docker - perconalab/pmm-client-fb:PR-811-77d2783
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-811-77d2783.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-811-8d4ef1f
client docker - perconalab/pmm-client-fb:PR-811-8d4ef1f
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-811-8d4ef1f.tar.gz

**Pull request merged**